e2e/tests/121-Portal/EditInfoPersonAffected/UpdateDateWithInvalidInput.spec.ts

```diff
@@ -0,0 +1,64 @@
+import HomePage from '@121-e2e/pages/Home/HomePage';
+import LoginPage from '@121-e2e/pages/Login/LoginPage';
+import RegistrationDetails from '@121-e2e/pages/RegistrationDetails/RegistrationDetailsPage';
+import TableModule from '@121-e2e/pages/Table/TableModule';
+import { SeedScript } from '@121-service/src/scripts/seed-script.enum';
+import ProgramTest from '@121-service/src/seed-data/program/program-test.json';
+import { importRegistrationsCSV } from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import { expect, test } from '@playwright/test';
+import englishTranslations from '../../../../interfaces/Portal/src/assets/i18n/en.json';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  await resetDB(SeedScript.test);
+  const programIdWesteros = 1;
+  accessToken = await getAccessToken();
+
+  await importRegistrationsCSV(
+    programIdWesteros,
+    './test-registration-data/test-registrations-OCW.csv',
+    accessToken,
+  );
+
+  // Login
+  const loginPage = new LoginPage(page);
+  await page.goto('/login');
```

Comment:
We have this enum:
https://github.com/global-121/121-platform/blob/main/interfaces/Portal/src/app/app-routes.enum.ts#L7

Maybe not for now, but we could 'all' be using the same set of URLs/paths?

Reply:
yes it is a good idea, I can put it as a point for the refactor.